src/main/java/ch/njol/skript/events/EvtEntityBlockChange.java

```diff
@@ -89,15 +91,15 @@ public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parser) {
 	}
 	
 	@Override
-	public boolean check(Event e) {
-		if (!(e instanceof EntityChangeBlockEvent))
+	public boolean check(Event event) {
+		if (!(event instanceof EntityChangeBlockEvent))
 			return false;
-		return event.checker.check((EntityChangeBlockEvent) e);
+		return this.event.checker.check((EntityChangeBlockEvent) event);
```

Comment:
"this" isn't needed as there is no field labeld event so it's misleading

Reply:
`private ChangeEvent event` at L85